From: Kevin Funk <kfunk@kde.org>
Date: Mon, 16 Jan 2017 09:40:05 +0100
Subject: CMake: Use SYSTEM for nm-glib/nm-util includes

---
 src/CMakeLists.txt | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index a8951ef..eb056a5 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -7,8 +7,10 @@ include_directories(
 else()
 include_directories(
    ${NETWORKMANAGER_INCLUDE_DIRS}
-   ${NM-UTIL_INCLUDE_DIRS}
-   ${NM-GLIB_INCLUDE_DIRS}
+
+   SYSTEM
+     ${NM-UTIL_INCLUDE_DIRS}
+     ${NM-GLIB_INCLUDE_DIRS}
 )
 endif()
 
@@ -143,7 +145,7 @@ set_target_properties(KF5NetworkManagerQt PROPERTIES VERSION ${NETWORKMANAGERQT_
 if (${NETWORKMANAGER_VERSION} VERSION_EQUAL 1.0.0 OR ${NETWORKMANAGER_VERSION} VERSION_GREATER 1.0.0)
     target_include_directories(KF5NetworkManagerQt PUBLIC ${NM-CORE_INCLUDE_DIRS})
 else()
-    target_include_directories(KF5NetworkManagerQt PUBLIC ${NETWORKMANAGER_INCLUDE_DIRS} ${NM-UTIL_INCLUDE_DIRS} ${NM-GLIB_INCLUDE_DIRS})
+    target_include_directories(KF5NetworkManagerQt PUBLIC ${NETWORKMANAGER_INCLUDE_DIRS} SYSTEM PUBLIC ${NM-UTIL_INCLUDE_DIRS} ${NM-GLIB_INCLUDE_DIRS})
 endif()
 
 ########### static lib for tests  ###############
