From: David Edmundson <kde@davidedmundson.co.uk>
Date: Thu, 5 Jan 2017 02:42:01 +0000
Subject: Initialise a member variable

REVIEW: 129777
---
 src/activeconnection.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/activeconnection.cpp b/src/activeconnection.cpp
index 22ab7d0..d797f6f 100644
--- a/src/activeconnection.cpp
+++ b/src/activeconnection.cpp
@@ -41,6 +41,7 @@ NetworkManager::ActiveConnectionPrivate::ActiveConnectionPrivate(const QString &
 #endif
     , dhcp4Config(0)
     , dhcp6Config(0)
+    , state(ActiveConnection::Unknown)
     , q_ptr(q)
 {
     path = dbusPath;
